src/lib.ts

```diff
@@ -31,6 +31,8 @@ class SingleFileServiceHost implements ts.LanguageServiceHost {
 	getScriptSnapshot = (name: string) => name === this.filename ? this.file : this.lib;
 	getCurrentDirectory = () => '';
 	getDefaultLibFileName = () => 'lib.d.ts';
+	fileExists = () => true;
+	readFile = () => '';
```

Comment:
Nice find! Just curious... should `readFile` return the actual contents of the file?

Reply:
According to my tests, `readFile` is used to load imported files, and `SingleFileServiceHost` seems to only analyze and process incoming content not imported files of incoming content. Maybe we can use `fallback` here like [gulp-typescript readFile](https://github.com/ivogabe/gulp-typescript/blob/b01326a77347520f1a24a877e3305ae272a1b432/lib/host.ts#L69), but I don't know the value of the `options` parameter.

Reply:
Thanks for the details!